### DISCUSSION TITLE
打包后小程序，真机调试，体验版都没问题，正式版白屏

### DISCUSSION THREAD

**Discussion by xuanwulb** [NervJS/taro#16608] [Q & A]
我的taro版本是 4.0.5，用的react写法，在真机调试，打包后上传体验版中均无问题，手机可以正常打开页面。但是线上的版本，手机端白屏，电脑端的微信打开没有问题。这种应该怎么解决啊

**Comment by TheKonka** [comment]
可以体验版打开 vconsole ，然后再打开 正式版本，这样也可以看到 vconsole，看看是不是报错了
